modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/fragment/renderer/ContentSectionFragmentRenderer.java

```diff
@@ -76,6 +85,12 @@ public void render(
 		throws IOException {
 
 		try {
+			ContentSectionDisplayContext contentSectionDisplayContext =
+				new ContentSectionDisplayContext(_cmsClassNamesConfiguration);
+
+			httpServletRequest.setAttribute(
+				WebKeys.PORTLET_DISPLAY_CONTEXT, contentSectionDisplayContext);
```

Comment:
@SamZiemer, to avoid conflicts with other attributes, it would be better yo use the className of the display context instead of  PORTLET_DISPLAY_CONTEXT

Reply:
Fixed in latest push